httpserver-rs/src/settings.rs

```diff
@@ -33,6 +33,7 @@ const CORS_ALLOWED_HEADERS: &[&str] = &[
     "accept-language",
     "content-type",
     "content-language",
+    "cache-control",
```

Comment:
Are these case-sensitive? I noticed we used `Cache-Control` on the response

Reply:
Headers are case insensitive